packages/plugin-zod/src/parser/index.ts

```diff
@@ -150,14 +150,25 @@ export function sort(items?: Schema[]): Schema[] {
   return transformers.orderBy(items, [(v) => order.indexOf(v.keyword)], ['asc'])
 }
 
+const shouldCoerce = (coercion: ParserOptions['coercion'] | undefined, type: 'dates' | 'strings' | 'numbers'): boolean => {
```

Comment:
Can you also a small test in https://github.com/kubb-project/kubb/blob/c8cb50d1e4a13669a05ca11a18352b86a558bce1/packages/plugin-zod/src/generators/zodGenerator.test.tsx

Appending something like this to the const testData will add another test case with those new options.
```
  {
      name: 'Pet',
      path: 'Pet',
      input: '../../mocks/petStore.yaml',
      options: {
           coercion: { ...}
     },
    },
```